holochain_test.go

```diff
@@ -9,10 +9,16 @@ import (
 	ic "github.com/libp2p/go-libp2p-crypto"
 	peer "github.com/libp2p/go-libp2p-peer"
 	. "github.com/smartystreets/goconvey/convey"
+	"os"
 	"testing"
 	"time"
 )
 
+func TestMain(m *testing.M) {
+	Initialize(InitializeProtocols)
+	os.Exit(m.Run())
+}
+
```

Comment:
This is the magic for that. It is just in the holochain package. I have to test if this location allows you to automatically have it with all individual tests and not just the ones in this file.